### PR TITLE
Ao 14676 graphql 2nd PR with a few tweaks

### PR DESCRIPTION
**Pull request by prettycoder**
- all execute layers are called graphql.execute with a :Key to specify
- only server errors are reported
- errors are reported as error events and not Errors KVs
- multiplex requests have a transaction name like graphql.multiplex.<operations>

- correction to case in AppOpticsAPM in grpc and some other files
- correction to the index args in the AppOpticsAPM.backtrace method

### CODE REVIEW

File: lib/appoptics_apm/inst/graphql.rb [hunk 1]
```diff
@@ -78,41 +78,40 @@ def gql_config
...
 
-          def maybe_set_transaction_name(query)
+          def multiplex_transaction_name(names)
             return if gql_config[:transaction_name] == false ||
                       ::AppOpticsAPM::SDK.get_transaction_name
 
-            split_query = query.strip.split(/\W+/, 3)
-            name = "graphql.#{split_query[0..1].join('.')}"
+            name = "graphql.multiplex.#{names.join('.')}"
+            name = "#{name[0..251]}..." if name.length > 254
```
Comment (molafson) [inline lib/appoptics_apm/inst/graphql.rb]: transaction name length would be 256 including the # and ..., should it be [0..250]?

Comment (prettycoder) [reply lib/appoptics_apm/inst/graphql.rb]: `#` is part of the expression operator `#{}`, 
e.g. `"three plus three is #{3+3}"` 
-> "three plus three is 3"

Comment (molafson) [reply lib/appoptics_apm/inst/graphql.rb]: okay, thanks for confirming, I didn't realize the # was part of the expression

**Comment by molafson** [comment]
Looks two commits look good.

**Pull request merged**